full-list.md

```diff
@@ -2,8 +2,12 @@
 
 ## Web browser extensions
 
-- `TTV LOL PRO` - [code](https://github.com/younesaassila/ttv-lol-pro)
-  - A fork of the `TTV LOL` extension with some UX improvements.
+- `TTV LOL PRO` - [chrome](https://chrome.google.com/webstore/detail/ttv-lol-pro/bpaoeijjlplfjbagceilcgbkcdjbomjd) / [code](https://github.com/younesaassila/ttv-lol-pro)
+  - A fork of the `TTV LOL` extension with sweeping improvements to its ad blocking abilities.
+  - Recommended to be used with uBlock Origin. Do not use with vaft/video-swap-new.
```

Comment:
The main readme already mentions to not combine Twitch specific ad blockers.

Reply:
Fixed.